Common/Source/Comm/device.cpp

```diff
@@ -598,45 +627,37 @@ BOOL devInit() {
             BitIndex = bit8N1;
             ReadPortSettings(i, Port, &SpeedIndex, &BitIndex);
         }
-        // remember: Port1 is the port used by device A, port1 may be Com3 or Com1 etc
-        if ((_tcsncmp(Port, _T("COM"),3)           == 0) ||
-            (_tcsncmp(Port, _T("IOIOUart_"), 9)    == 0) ||
-            (_tcsncmp(Port, _T("USB:"), 4)         == 0) ||
-            (_tcscmp(Port, _T("Bluetooth Server")) == 0)
-           )
-        {  // shared ports for COM Ports only
-          if(std::find(UsedPort.begin(), UsedPort.end(), Port) != UsedPort.end()) {
-            unsigned int j;
-            for( j = 0; j < i ; j++)
-            {
-              if(!DeviceList[i].Disabled)
-              if( (_tcscmp(szPort[i] , szPort[j])==0)  &&  DeviceList[j].iSharedPort <0)
-              {
-                devInit(&DeviceList[i]);
-                DeviceList[i].iSharedPort =j;
-                StartupStore(_T(". Port <%s> Already used, Device %c shares it with %c ! %s"), Port, (_T('A') + i),(_T('A') + j), NEWLINE);
-                DeviceList[i].Com = DeviceList[j].Com ;
-                DeviceList[i].Status = CPS_OPENOK;
-                pDev->Installer(&DeviceList[i]);
-                if (pDev->Flags & (1l << dfNmeaOut)) {
-                    DeviceList[i].bNMEAOut = true;
-                }
-                if(devIsRadio(&DeviceList[i]))
-                {
-                  RadioPara.Enabled = true;
-                  StartupStore(_T(".  RADIO  %c  over  <%s>%s"), (_T('A') + i),  Port, NEWLINE);
-                }
-              }
-            }
-            continue;
-          }
-        }
-        UsedPort.insert(Port);
-        
-        // remember: Port1 is the port used by device A, port1 may be Com3 or Com1 etc
+
+	unsigned int j;
+	DeviceList[i].iSharedPort =-1;
+	for( j = 0; j < i ; j++)
+	{
```

Comment:
"j" is local loop iterator so no need to declare them outside

it's better to write 

`for(unsigned j = 0; j < i ; j++)`

Reply:
done:
https://github.com/LK8000/LK8000/pull/1328